data/categories.csv

```diff
@@ -18,6 +18,7 @@ movies,Movies
 music,Music
 news,News
 outdoor,Outdoor
+politics,Politics

```

Comment:
```suggestion
public-affairs,Public affairs
```

Reply:
I don't know if it is possible to use hyphens or spaces for two words, but I leave the proposal here.

Reply:
I would say that an underscore would be more fitting, rather than a hyphen.